aidial_adapter_openai/gpt4_multi_modal/chat_completion.py

```diff
@@ -111,6 +119,20 @@ async def predict_non_stream(
             return await response.json()
 
 
+def multimodal_truncate_prompt(
+    transformations: List[MultiModalMessage],
```

Comment:
`transformations` here doesn't make sense. 
We don't really care about the past of this object, we care about what it actually represents now. I suggest simply `messages`.

Reply:
fixed